service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/LegalRepresentativesChangeToCaseEventHandlerTest.java

```diff
@@ -0,0 +1,134 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.fpl.events.LegalRepresentativesChangeToCaseEvent;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.LegalRepresentative;
+import uk.gov.hmcts.reform.fpl.model.LegalRepresentativesChange;
+import uk.gov.hmcts.reform.fpl.service.LegalRepresentativesDifferenceCalculator;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.LegalRepresentativeAddedContentProvider;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static java.util.Collections.emptySet;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.LEGAL_REPRESENTATIVE_ADDED_TO_CASE_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+@ExtendWith(MockitoExtension.class)
+class LegalRepresentativesChangeToCaseEventHandlerTest {
+
+    public static final List<LegalRepresentative> LEGAL_REPRESENTATIVES_BEFORE =
+        List.of(mock(LegalRepresentative.class));
+    public static final CaseData CASE_DATA_BEFORE = CaseData.builder()
+        .legalRepresentatives(wrapElements(LEGAL_REPRESENTATIVES_BEFORE))
+        .build();
+    public static final List<LegalRepresentative> LEGAL_REPRESENTATIVES_NOW = List.of(mock(LegalRepresentative.class));
```

Comment:
is there a reason why you are using a mock here? and above?

Reply:
LEGAL_REPRESENTATIVES_NOW is a pass parameter mock, so the service that receives hat as a input just cares that the instance is exactly the one instructed.